guides/tutorials/registering-schema-and-credential-definition-on-indy.md

```diff
@@ -0,0 +1,66 @@
+# Registering a schema and credential definition on an Hyperledger Indy ledger
+
+In this tutorial we will register a schema and credential definition on an Hyperledger Indy ledger using [indy-vdr](https://github.com/hyperledger/indy-vdr).
+
+:::info
+
+This section assumes that
+
+1. You have [set-up your develoment environment](../getting-started).
+1. You have basic knowledge of the required fields in the [Agent Config](./agent-config)
+
+:::
+
+### 1. Setting up the agent
+
+First, an agent must be setup with a wallet and an indy vdr module. For these code examples, we will use a Node.js environment.
+
+:::issuer
+
+```typescript showLineNumbers register-schema-and-cred-def.ts section-1
+
+```
+
+:::
+
+### 2. Importing a DID (optional)
+
+In order to register a schema and credential definition, a DID must be added to the agent first. This can be done by caling `agent.dids.create()`, but this does require an endorser DID to be present as the `submitterDid`. For this tutorial a DID will already be registered on the ledger, but it will be imported, whith the associated private key, to be used later when registering the schema and credential definition.
```

Comment:
Maybe we can add a info/tip section that shows how to do this with bcovrin test. E.g. you can register your did on https://test.bcovrin.vonx.io. That way they can follow along with the tutorial, but they can also use their own network if they want.